### PR TITLE
init/ezid: fetch credentials from secrets.yml

### PR DESCRIPTION
**Pull request by dunn**
Depends on https://github.com/acozine/sufia-centos/pull/70

Currently when Ezid credentials are stored in `secrets.yml`, ingests fail with this error:
```
/usr/local/lib/ruby/2.2.0/uri/generic.rb:767:in `check_path': bad
component(expected absolute path component): /shoulder/<%=
Rails.application.secrets.ezid_default_shoulder %>
(URI::InvalidComponentError)
  from /usr/local/lib/ruby/2.2.0/uri/generic.rb:815:in `path='
  from /usr/local/lib/ruby/2.2.0/uri/generic.rb:189:in `initialize'
  from /usr/local/lib/ruby/2.2.0/uri/http.rb:84:in `initialize'
  from /usr/local/lib/ruby/2.2.0/uri/generic.rb:134:in `new'
  from /usr/local/lib/ruby/2.2.0/uri/generic.rb:134:in `build'
  from /usr/local/lib/ruby/2.2.0/uri/http.rb:62:in `build'
  from
/opt/alexandria-v2/shared/bundle/ruby/2.2.0/gems/ezid-client-1.2.0/lib/ezid/requests/request.rb:110:in
`build_uri'
```

I'm sure there's a way to fix that, but since the credentials are in `secrets.yml` after all we may as well fetch them directly.

cc @acozine

### CODE REVIEW

**Comment by jcoyne** [comment]
The tests are failing.  I think you need to move these lines from config/ezid.yml.template to config/secrets.yml.template:
```
  user:             apitest
  password:         ez_id_password
```

**Comment by jcoyne** [comment]
While you're working on this, I'm wondering if it makes sense to get rid of ezid.yml and put the host/port in config/application.yml?

**Comment by dunn** [comment]
> I'm wondering if it makes sense to get rid of ezid.yml and put the host/port in config/application.yml?

Sounds good!

File: config/secrets.yml.template [hunk 1]
```diff
@@ -10,17 +10,20 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
-development: &dev
+default: &default
```
Comment (jcoyne) [inline config/secrets.yml.template]: Please add a development entry such that it just works, when you copy secrets.yml.template to config/secrets.yml and start working in development mode.

Comment (dunn) [inline config/secrets.yml.template]: Ah, woops. Fixed.

**Pull request merged**